meson.build

```diff
@@ -1,51 +1,42 @@
-project('com.github.rajkumaar23.badgie', 'vala', 'c',
+# Project name, programming language and version
+project (
+    'com.github.rajkumaar23.badgie',
+    'vala', 'c',
     version: '0.1.4'
 )
 
-gnome = import('gnome')
-i18n = import('i18n')
 
-add_project_arguments(
-    '-DGETTEXT_PACKAGE="@0@"'.format(meson.project_name()),
+# Translation module
+i18n = import ('i18n')
+
+# Gnome module
+gnome = import ('gnome')
+
+# Project arguments
+add_project_arguments (
+    '-DGETTEXT_PACKAGE="@0@"'.format (meson.project_name ()),
     language: 'c'
 )
 
-executable(
-    meson.project_name(),
-    'src/Application.vala',
-    'src/MainWindow.vala',
-    dependencies: [
-        dependency('gtk+-3.0'),
-        dependency('libsoup-2.4'),
-        dependency('gee-0.8'),
-        dependency('gdk-pixbuf-2.0')
-    ],
-    install : true
-)
+# Listing dependencies
+dependencies = [
+    dependency ('glib-2.0'),
```

Comment:
Why is this new dependency introduced ?

Reply:
It's often used as a default dependency like granite or gtk. It's usefull to handle translation

Reply:
Okay. Thanks for the PR :)